taigit/src/libraries/Taiga.ts

```diff
@@ -877,4 +877,45 @@ check_for_retrospective(project_id : number, sprint_id : number) : Promise<wiki_
         }
     }
     return retro_page;
-}
\ No newline at end of file
+}
+
+
+
+**
+ * @summary This call return estimate of sprint plan need to finish the project (unit points) based on project Id
+ * @param projectId the ID for the access of project
+ * @returns
+ *  * Array of sprint plan [sprint_num, velocity]
+ * * {
+ *          sprint_num : number // num of sprint needed
+ *          velocity : number //estimate sprint velocity
+ * }
+ */
+
+export async function
+proj_spplan_res(projId : number) : Promise<Array<[number,number]>> {
+    let velocity_list : Array<[boolean, number]> =  (await proj_spvelocity(projId));
+    //test case//286226//306316
+    //calculate sprint velocity stats
+    let v_min : number = Number.MAX_VALUE;
+    let v_max : number = 0;
+    let v_avg : number = 0;
+    let v_sum : number = 0;
+    let n_s : number = 0;
+    for(let v of velocity_list){
+        if(v[0]) {
+            if (v[1] < v_min)
+                 v_min = v[1];
+             if (v[1] > v_max)
+                 v_max = v[1];
+             v_sum += v[1];
+             n_s += 1;
+         }
+
+    }
+    v_avg=v_sum/n_s;
+    //estimate future sprint planning
+    let project = (await axios.get("https://api.taiga.io/api/v1/projects/" + projId.toString() + '/stats')).data;
```

Comment:
`project_stats(projId)`

Reply:
i have issue using project_stats.

Reply:
maybe you could ask for help about said issue?

Reply:
let project = (await project_stats(projId)).data;
if this works on your side, i can change the line.